gulp/tasks/test.js

```diff
@@ -93,6 +94,36 @@ function _getProtractorParams() {
   return params;
 }
 
+gulp.task( 'test:acceptance:perf', sitespeedio( {
```

Comment:
This can be top level in the testing task I'd say `test:perf`, like `test:a11y`.